doc/getting_started/installation_win.rst

```diff
@@ -196,13 +196,12 @@ to set it up:
 
 You're now almost ready to continue with the rest of the getting started guide.
 
-Since you're using MSYS2, when you're cloning Zephyr in the next step of the
-guide, use this command line instead (i.e. add the ``--config
-core.autocrlf=false`` option).
+Since you're using MSYS2, before you clone Zephyr set the ``autocrlf`` option
```

Comment:
Another PR (#11300) is removing MSYS2 support (and documentation), so we shouldn't mention MSYS2 here.